README.md

```diff
@@ -90,214 +58,220 @@ type: tutorial
 
 ---
 
-# Store and query unstructured JSON data from CockroachDB hosted on Red Hat Marketplace
-
-In this tutorial, we will learn how to get unstructured JSON data from an API, store it in CockroachDB hosted on Red Hat Marketplace, and query the unstructured JSON data from the table using python runtime and Jupyter notebook.
+This tutorial shows you how to get unstructured JSON data from an API, store it in CockroachDB hosted on Red Hat Marketplace, and query the unstructured JSON data from the table using Python and Jupyter notebooks.
 
 # About CockroachDB Operator
 
-CockroachDB is a ultra resilient, distributed SQL that can easily scale-out serializable transactions for your apps and services. It is cloud-native, architected to simplify scale and also guarantee consistent transactions across multiple regions and multiple clouds. [Learn more](https://marketplace.redhat.com/en-us/products/cockroachdb-operator).
+CockroachDB is an elastic SQL database that easily scales transactions for your apps and services. Its cloud-native architecture simplifies scale and also guarantees consistent transactions across multiple regions and multiple clouds. [Learn more](https://marketplace.redhat.com/en-us/products/cockroachdb-operator).
 
 # Learning objectives
 
 When you have completed this tutorial, you will understand how to:
 
-* Install CockroachDB Operator from Red Hat Marketplace on a OpenShift Cluster
+* Install CockroachDB Operator from Red Hat Marketplace on an OpenShift Cluster
 * Create a CockroachDB cluster instance
 * Create a user and database in CockroachDB
 * Store and query unstructured JSON data from a third-party API in CockroachDB
-* Manage CockroachDB cluster from the Cluster Overview page
+* Manage the CockroachDB cluster from the Cluster Overview page
 
 # Estimated time
 
 Completing this tutorial should take about 30 minutes.
 
 # Pre-requisites
 
-1. [Red Hat Marketplace Account](https://marketplace.redhat.com/en-us/registration/om).
-2. [Red Hat OpenShift Cluster](https://cloud.ibm.com/kubernetes/catalog/create?platformType=openshift). 
-3. [OC & kubectl CLI](https://docs.openshift.com/container-platform/3.6/cli_reference/get_started_cli.html).
+1. [Red Hat Marketplace account](https://marketplace.redhat.com/en-us/registration/om).
+2. [Red Hat OpenShift Cluster](https://cloud.ibm.com/kubernetes/catalog/create?platformType=openshift). <--EM: this is different than the doc we referenced in the Get Started tutorial we just did. We linked here: https://cloud.ibm.com/docs/openshift?topic=openshift-getting-started when telling readers to set up a RH cluster-->
+3. [OC & kubectl CLI](https://docs.openshift.com/container-platform/3.6/cli_reference/get_started_cli.html).<!--EM: this link is outdated. Is it okay to point here? And what does "OC" stand for? https://docs.openshift.com/container-platform/3.11/welcome/index.html-->
```

Comment:
Oh, I see, here is the updated link you can replace it with this one: <https://docs.openshift.com/container-platform/4.5/welcome/index.html> and OC stand for OpenShift Container.

Reply:
@manojjahgirdar Okay, do you think your 4.5 link is better than the 3.11 link? I noticed that  the 4.5 one doesn't show IBM Cloud, so I was curious if IBM Cloud doesn't support 4.5?

Reply:
Okay! I wasn't sure whether to point to 3.11 or 4.5 b/c I didn't see IBM Cloud on the 4.5 page. I wasn't sure if it was supported or if that would mess up how things work?

Reply:
We Just need the OC CLI part from the URL, to be more specific <https://docs.openshift.com/container-platform/4.5/cli_reference/openshift_cli/getting-started-cli.html> this is the link for 4.5.